pkg/clusteragent/clusterchecks/types/types.go

```diff
@@ -58,10 +58,11 @@ type Stats struct {
 // need and allows to inject a custom one for tests
 type LeaderIPCallback func() (string, error)
 
-// EndpointsInfo is used to store the collected info of endpoints.
```

Comment:
Why is this removed ?

Reply:
the struct isn't used anymore, I should have cleaned it during the Endpoints check feature revamp